diff --git a/ios/DevMenuAppInstance.swift b/ios/DevMenuAppInstance.swift
index a13e02916dac17fbd94961a92dc9ec303f7e4e07..196f8689262caab2e96fc4a392456c2643ca0a05 100644
--- a/ios/DevMenuAppInstance.swift
+++ b/ios/DevMenuAppInstance.swift
@@ -52,7 +52,7 @@ class DevMenuAppInstance: DevMenuRCTAppDelegate {
     return jsSourceUrl()
   }
 
-  override func extraModules(for bridge: RCTBridge!) -> [RCTBridgeModule]! {
+  override func extraModules(for bridge: RCTBridge!) -> [RCTBridgeModule] {
     var modules: [RCTBridgeModule] = [DevMenuLoadingView.init()]
     modules.append(DevMenuRCTDevSettings.init())
     return modules
diff --git a/ios/ReactDelegateHandler/ExpoDevMenuReactDelegateHandler.swift b/ios/ReactDelegateHandler/ExpoDevMenuReactDelegateHandler.swift
index 440902c134d9f2a2c6bf0486f880df5a60bdbdfc..9ba4ab96ed165cf8239e80d68d03ee94c86d510c 100644
--- a/ios/ReactDelegateHandler/ExpoDevMenuReactDelegateHandler.swift
+++ b/ios/ReactDelegateHandler/ExpoDevMenuReactDelegateHandler.swift
@@ -21,7 +21,7 @@ public class ExpoDevMenuReactDelegateHandler: ExpoReactDelegateHandler {
     return true
   }()
 
-  public override func createRootView(reactDelegate: ExpoReactDelegate, bridge: RCTBridge, moduleName: String, initialProperties: [AnyHashable : Any]?) -> RCTRootView? {
+  public func createRootView(reactDelegate: ExpoReactDelegate, bridge: RCTBridge, moduleName: String, initialProperties: [AnyHashable : Any]?) -> RCTRootView? {
     if ExpoDevMenuReactDelegateHandler.shouldEnableAutoSetup {
       DevMenuManager.shared.currentBridge = bridge
     }
